packages/mds-db/events.ts

```diff
@@ -334,10 +334,10 @@ export async function readEventsWithTelemetry({
   const { rows } = await exec(
     `SELECT E.*, T.lat, T.lng, T.speed, T.heading, T.accuracy, T.altitude, T.charge, T.timestamp AS telemetry_timestamp FROM (SELECT * FROM ${
       schema.TABLE.events
-    }${where} ORDER BY id LIMIT ${vals.add(limit)}
+    }${where} ORDER BY timestamp DESC LIMIT ${vals.add(limit)}
     ) AS E LEFT JOIN ${
       schema.TABLE.telemetry
-    } T ON E.device_id = T.device_id AND CASE WHEN E.telemetry_timestamp IS NULL THEN E.timestamp ELSE E.telemetry_timestamp END = T.timestamp ORDER BY id`,
+    } T ON E.device_id = T.device_id AND CASE WHEN E.telemetry_timestamp IS NULL THEN E.timestamp ELSE E.telemetry_timestamp END = T.timestamp ORDER BY timestamp DESC`,
```

Comment:
This change will **break** the MDS Native `/events` API which uses the `id` column as a `polling` cursor. I see you have a couple of alternatives:
1. Allow the caller to override the `ORDER BY` clause (with the default being `ORDER BY id`)
2. Always `ORDER BY id` and then sort the results in `MDS Audit`
3. Create a separate method for audit.

Reply:
Good catch, thanks